WORKSPACE

```diff
@@ -0,0 +1,22 @@
+workspace(name = "grpc_example")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "com_github_grpc_grpc",
+    urls = [
+        "https://github.com/grpc/grpc/archive/4d3f0604eb5e20e94a01a1de2ac6ca27ab41e180.tar.gz",
```

Comment:
Why is this referring to a hash, not the released version?

Reply:
fixed